src/js/views/Vote.vue

```diff
@@ -78,7 +78,7 @@
 
 		<div v-if="poll.anonymous" class="area__footer">
 			<div>
-				{{ t('poll', 'Although participant\'s names are hidden, this is no real anonymous poll, bacause they are not hidden for the owner.') }}
+				{{ t('poll', 'Although participants names are hidden, this is not a real anonymous poll because they are not hidden from the owner.') }}
```

Comment:
should still be genitive (participant's)

Reply:
Corrected 😉.